decidim-proposals/app/helpers/decidim/proposals/admin/proposals_helper.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Proposals
+    module Admin
+      # This class contains helpers needed to format Meetings
+      # in order to use them in select forms for Proposals.
+      #
+      module ProposalsHelper
+        # Public: A formatted collection of Meetings to be used
+        # in forms.
+        def meetings_selected
```

Comment:
The name of the method is misleading since it returns the authors ids, not meetings.